app/models/cart.rb

```diff
@@ -1,28 +0,0 @@
-class Cart < ActiveRecord::Base
```

Comment:
I think removing the model should also go with a migration to remove the table. It also needs to drop the relationship between order and cart.

I just checked our production server for carts and our table is empty. So this should be safe.

I found several uses of `order.cart?`, but I suspect that it's testing whether the order is in the `cart` state and not if it has a cart. Can you double check that?

Reply:
I doubled checked, yes, cart is one of the states of the order state machine.
All I have found on my double checking was more dead code, please see the new commit.